app/models/role.rb

```diff
@@ -0,0 +1,3 @@
+class Role < ApplicationRecord
+    validates :name, presence: true
```

Comment:
Deberías configurar la tabulación en tu editor de código a 2 espacios en vez de 4:
```suggestion
  validates :name, presence: true
```